src/labels/CustomLabels.labels

```diff
@@ -5013,6 +5013,22 @@ To check your GAU Allocation settings, go to Donations | GAU Allocations.</value
         <shortDescription>pmtModifyCurrency</shortDescription>
         <value>You can't modify Payment currencies directly. Update the parent Opportunity's currency, and NPSP will automatically update the currency of all related Payment records.</value>
     </labels>
+    <labels>
+        <fullName>pmtPaidPaymentDateRequired</fullName>
```

Comment:
@jbrandolinisf These are the new custom labels, any feedback on the name/categories?

Reply:
Looks good to me. Will defer to any desired changes on the part of @jbrandolinisf